src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/PartATemplateReplacementServiceTest.kt

```diff
@@ -160,6 +225,7 @@ internal class PartATemplateReplacementServiceTest : ServiceTestBase() {
       assertThat(result["phone_number"]).isEqualTo("555-0100")
       assertThat(result["fax_number"]).isEqualTo("555-0199")
       assertThat(result["email_address"]).isEqualTo("thomas.magnum@gmail.com")
+      assertThat(result["email_address"]).isEqualTo("thomas.magnum@gmail.com") // TODO - BS how do I handle checkboxes
```

Comment:
Is the duplicate assertion here intended?

Reply:
removed :)